SMUser/schema-LD.json

```diff
@@ -0,0 +1,59 @@
+{

```

Comment:
possibly not necessary. See previous comments regarding this topic.

Reply:
This might be true, see my previous replies.